File: jutopia/teacher/src/main/java/com/ssafy/teacher/dto/rent/Seat.java
package com.ssafy.teacher.dto.rent;

import lombok.Getter;
import lombok.Setter;

import java.util.List;


@Getter
@Setter
public class Seat {
    private String id;
    private int position;
    private int price;
    private String user_id;
    private String seat_status;
    private int clazz_number;
    private int grade;
    private String school;

}

File: jutopia_kmm/shared/src/commonMain/kotlin/lease/Seat.kt
package lease

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

//data class Seat(val id: Int, val info: String, val price: Int, val isAvailable: Boolean)


@Serializable
data class ListResponse(
    val result: ApiResult,
    val body: List<Seat>?
)

@Serializable
data class Response(
    val result: ApiResult,
    val body: Seat?
)


@Serializable
data class ApiResult(
    @SerialName("result_code") val resultCode: Int,
    @SerialName("result_message") val resultMessage: String,
    @SerialName("result_description") val resultDescription: String
)
@Serializable
data class Seat(
    val id: String,
    val position: Int,
    val price: Double,
    @SerialName("user_id") val userId: String?,
    @SerialName("seat_status") val seatStatus: String,
    @SerialName("clazz_number") val clazzNumber: Int,
    val grade: Int,
    val school: String,
)

@Serializable
data class SeatRequest(
    @SerialName("seat_id")val seatId: String,
    @SerialName("user_id")val userId: String,
)